packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTable.tsx

```diff
@@ -235,18 +229,6 @@ export function TransactionsTable({
     return [...newerTransactions.reverse(), ...subgraphTransactions]
   }, [transactions, localTransactionsKey])
```

Comment:
We haven't had them before but I will have a look.

Maybe @dewanshparashar can have a look as well as I don't believe I've made any changes here before.

Reply:
In dependencies, 
- we don't need `locallyStoredTransactions` since this state object mutates very frequently and `localTransactionsKey (string)` is derived from it to keep things under control.
- `pageNumber` and `searchString` can be added, but we only need this memo function to run for the very first page so adding these doesn't make a difference to the logic / result. Just that it will recalculate and do an early return for each update. It's optional.

If we go ahead with it, I'd suggest a different PR to isolate it's impact.

Reply:
> we don't need locallyStoredTransactions since this state object mutates very frequently and localTransactionsKey (string) is derived from it to keep things under control.

Then we should either memo it, or not having it as a dependencies, otherwise it make things unpredictable

> pageNumber and searchString can be added, but we only need this memo function to run for the very first page so adding these doesn't make a difference to the logic / result. Just that it will recalculate and do an early return for each update. It's optional.

If they are part of the callback of useEffect, then they should be included, even if they never change